src/styles/switch.scss

```diff
@@ -0,0 +1,74 @@
+// ----- Switch Style @radix-ui ----- 
+button {
+    all: unset;
+}
+
+.SwitchRoot {
+    width: 42px;
+    height: 25px;
+    background-color: #A9A9A9;
+    border-radius: 9999px;
+    position: relative;
+    box-shadow: 0 2px 10px #A7A7A7;
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+.SwitchRoot:focus {
+    box-shadow: 0 0 0 2px #00BFFF;
+}
+
+.SwitchRoot[data-state='checked'] {
+    background-color: #00BFFF;
+}
+
+.SwitchThumb {
+    display: block;
+    width: 21px;
+    height: 21px;
+    background-color: white;
+    border-radius: 9999px;
+    box-shadow: 0 2px 2px var(--black-a7);
+    transition: transform 100ms;
+    transform: translateX(2px);
+    will-change: transform;
+}
+
+.SwitchThumb[data-state='checked'] {
+    transform: translateX(19px);
+}
+
+.Label {
+    color: black;
+    font-size: 15px;
+    line-height: 1;
+}
+
+// ----- End Switch Style @radix-ui ----- 
+
+
+.submit-btn {
+    display: inline-block;
+    padding: 10px 20px;
+    border: none;
+    border-radius: 15px;
+    background-color: #007bff;
+    color: #fff;
+    font-size: 16px;
+    text-align: center;
+    text-decoration: none;
+    cursor: pointer;
+    transition: background-color 0.3s ease;
+
+    &:hover {
+        background-color: #0056b3;
+    }
+
+    &:focus {
+        outline: none;
+        box-shadow: 0 0 0 3px rgba(0, 123, 255, 0.3);
+    }
+
+    &:active {
+        background-color: #004180;
+    }
+}
```

Comment:
please extract these to another file :) For now, we can put this under `components/TextGenerationForm.scss` (btw a type on folder name, `components` not `components`) because the button is in this component. Later we're like to have a `sass/component/button.scss` once we have the design system ready. This `switch.scss` should have a single responsibility which is to style the radix-ui's Switch. For the sass folder structure, please refer to this https://sass-guidelin.es/#the-7-1-pattern